.github/actions/k-bench/evaluate/graph.py

```diff
@@ -0,0 +1,181 @@
+"""Generate graphs comparing K-Bench benchmarks across cloud providers and Constellation."""
+import json
+import os
+from collections import defaultdict
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+SUBJECTS = [
+    "constellation-gcp",
+    "GKE",
+    "constellation-azure",
+    "AKS",
```

Comment:
```suggestion
    "Constellation on GCP",
    "GKE",
    "Constellation on Azure",
    "AKS",
```

Or similar. I might even suggest switching the order of GCP and Azure for consistency in our docs but if it's too much effort it's not necessary I guess.

Reply:
Sure. Since this "constellation-$CSP" format is also used for the directory structure it is a bit messy to change it there and introduce spaces.

Therefore, I suggest just adding a legend list that the graph generator uses. Changing the order shouldn't be of concern though.